README.md

```diff
@@ -385,6 +387,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 *Libraries which provide general utility functions.*
 
+* [args4j](http://args4j.kohsuke.org/) - Command line arguments parser
```

Comment:
Yeah, I have pushed this one after creating pull request, didn't know it will end up here.